docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md

```diff
@@ -101,7 +101,7 @@ actions.perform()
 # We don't have a Ruby code sample yet -  Help us out and raise a PR
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR
+ await actions.move({origin: e}).perform();
```

Comment:
Let's try to be consistent with examples of other languages with a complete syntax. Would the below help?
await driver.actions()
          .move({origin: e})
          .perform();

Reply:
HI @manoj9788 ,

Yes, as the above code has the driver declaration 'const actions = driver.actions();'. i have updated code as
 await actions
 .move({origin: e})
 .perform();